week2/week2/Program.cs

```diff
@@ -0,0 +1,256 @@
+﻿// See https://aka.ms/new-console-template for more information
+//Exercise 1
+using System.Diagnostics.Metrics;
+using System.Reflection.Metadata.Ecma335;
```

Comment:
For sure this two lines are not necessary and can be removed. You'll learn what they are a bit later.